src/sentry/static/sentry/app/views/organizationDiscover/result/chart.jsx

```diff
@@ -1,18 +1,106 @@
 import React from 'react';
+import moment from 'moment';
 import PropTypes from 'prop-types';
 
-export default class ResultChart extends React.Component {
+import BarChart from 'app/components/charts/barChart.jsx';
```

Comment:
Damn I thought we had a lint rule but apparently it's `TODO`: https://github.com/getsentry/eslint-config-sentry/blob/master/packages/eslint-config-sentry-react/rules/imports.js#L74

We should follow the other imports though and leave out file extensions in imports.